manager/controllers/systems_advisories_view.go

```diff
@@ -0,0 +1,68 @@
+package controllers
+
+import (
+	"app/base/database"
+	"app/manager/middlewares"
+	"github.com/gin-gonic/gin"
+	"github.com/jinzhu/gorm"
+	"net/http"
+)
+
+type SystemsAdvisoriesRequest struct {
+	Systems    []string `json:"systems"`
+	Advisories []string `json:"advisories"`
+}
+
+type SystemsAdvisoriesResponse struct {
+	Data map[string][]string `json:"data"`
+}
+
+type systemsAdvisoriesDBLoad struct {
+	SystemID   string `query:"sp.inventory_id"`
+	AdvisoryID string `query:"am.name"`
+}
+
+var systemsAdvisoriesSelect = database.MustGetSelect(&systemsAdvisoriesDBLoad{})
+
+func systemsAdvisoriesQuery(acc int, systems, advisories []string) *gorm.DB {
+	return database.Db.
+		Table("system_advisories sa").
+		Select(systemsAdvisoriesSelect).
+		Joins("join system_platform sp on sp.rh_account_id = ? and sp.id = sa.system_id", acc).
+		Joins("join advisory_metadata am on am.id = sa.advisory_id").
+		Where("sa.rh_account_id = ?", acc).
+		Where("sp.inventory_id in (?)", systems).
+		Where("am.name in (?)", advisories).
+		Order("sp.inventory_id, am.id")
```

Comment:
We should conditionally use "inventory.hosts" table to filter system_platform items as used in other handlers.

Reply:
Added